tests/streaming_test.py

```diff
@@ -523,6 +523,30 @@ async def test_account_activity_subs_success(self, ws_connect):
             }
         })
 
+    @no_duplicates
+    @asynctest.patch('tda.streaming.ws_client.connect', new_callable=asynctest.CoroutineMock)
+    async def test_account_activity_unsubs_success(self, ws_connect):
+        socket = await self.login_and_get_socket(ws_connect)
+
+        socket.recv.side_effect = [json.dumps(self.success_response(
+            1, 'ACCT_ACTIVITY', 'UNSUBS'))]
```

Comment:
That being said, let's investigate what TDA does when you login in and immediately unsub from a stream to which you're not already subbed. My gut tells me this is 1. something people will probably do by mistake and 2. a potentially interesting corner case that we don't understand and should test.